pages/common/ogr2ogr.md

```diff
@@ -1,24 +1,24 @@
 # ogr2ogr
 
-> Converts simple features data between file formats.
+> Convert simple features data between file formats.
```

Comment:
What does "features data" mean in this context? It doesn't make grammatical sense.

Reply:
I think it's the kind of data that these file formats contain: https://gdal.orgprograms/ogr2ogr.html it sounds strangely generic, so basically just how such kind of data is called 🤔🤔🤔

Reply:
[Simple Features](https://en.wikipedia.org/wiki/Simple_Features) are a specification how to store geographic data. A "feature" can be a point, line or polygon. It is a well known phrase in the domain of geodata. Let me know if I shall clarify more.

Reply:
Ah, I see. In that case, I think it would make sense to capitalise it.
```suggestion
> Convert Simple Features data between file formats.
```

Reply:
done in bedb542cf10d8748cf130e7d72ccb0fb4631e43a